spec/services/proofing/aamva/applicant_spec.rb

```diff
@@ -69,7 +69,8 @@
     proofer_applicant[:height] = 73
```

Comment:
I feel annoying asking this, but how would you feel about making this 67, so that the test below matches the example?